contrib/packaging/rpm/Makefile

```diff
@@ -1,18 +1,17 @@
 include ../../../Makefile.defs
 
-BUILDDIR := "$(CURDIR)/cilium-$(VERSION)"
+CWD := $(CURDIR)
+VERSION := 0.1.0
```

Comment:
Can you read the version from [here](https://github.com/cilium/cilium/blob/master/VERSION)?